src/main/java/edu/ie3/datamodel/io/processor/EntityProcessor.java

```diff
@@ -37,10 +43,20 @@
   public final Logger log = LogManager.getLogger(this.getClass());
   private final Class<? extends T> registeredClass;
   protected final String[] headerElements;
-  protected final LinkedHashMap<String, Method> fieldNameToMethod = new LinkedHashMap<>();
+  private final Map<String, Method> fieldNameToMethod;
+
+  private static final String OPERATION_TIME_FIELD_NAME = OperationTime.class.getSimpleName();
+  private static final String OPERATES_FROM = "operatesFrom";
+  private static final String OPERATES_UNTIL = "operatesUntil";
+
+  private static final String VOLT_LVL_FIELD_NAME = "voltLvl";
+  private static final String VOLT_LVL = NodeInputFactory.VOLT_LVL;
+  private static final String V_RATED = NodeInputFactory.V_RATED;
+
+  private static final GeoJsonWriter geoJsonWriter = new GeoJsonWriter();
 
   /** Field name of {@link UniqueEntity} uuid */
-  private static final String uuidString = "uuid";
+  private static final String UUID_FIELD_NAME = "uuid";
```

Comment:
because it directly refers to the field/attribute name in a entity we are processing, while in contrast all other values without the `_FIELD_NAME` suffix are "derived fields" in the sense that they are not fields in our main object instance we are processing but wrapped in e.g. `OperationTime` or `VoltLevel` instances - hence `field` in this case is the real field in the entity